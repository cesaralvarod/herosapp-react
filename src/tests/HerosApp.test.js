import React from "react";
import { shallow } from "enzyme";
import HerosApp from "../HerosApp";

describe("Testing in HerosApp.js", () => {
  test("Testing in component <HerosApp />", () => {
    const wrapper = shallow(<HerosApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
