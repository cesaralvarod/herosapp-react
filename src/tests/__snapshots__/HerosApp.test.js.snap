// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing in HerosApp.js Testing in component <HerosApp /> 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "user": Object {
        "logged": false,
      },
    }
  }
>
  <AppRouter />
</ContextProvider>
`;
